.markdownlint-cli2.yaml

```diff
@@ -0,0 +1,11 @@
+#  Configuration for the markdownlint-cli2, see:
```

Comment:
I'm curious: Could you explain in the commit message why you chose https://github.com/DavidAnson/markdownlint over https://github.com/markdownlint/markdownlint?

Reply:
The main reason was that it can be run with `npx` which is already required for `commitlint`, while the other linter requires Ruby. I have added more information to the commit message.

Reply:
Btw, I did try the former first, but I did not do a proper comparison of the results.